service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -537,5 +504,214 @@
     "tags":[],
     "targetableText":"I talked to Santa Claus, and he says it's not within his power to bring [TARGET] a real friend for Christmas.",
     "targetableSsml":"I talked to Santa Claus, and he says it's not within his power to bring [TARGET] a real friend for Christmas."
+  },
+  {
+    "quipGroup":"QUITE_SIMPLE",
+    "text":"If someone were to ask me to describe you, it would be, quite simple.",
+    "ssml":"If someone were to ask me to describe you, it would be, quite simple.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"If someone were to ask me to describe [TARGET], it would be, quite simple.",
+    "targetableSsml":"If someone were to ask me to describe [TARGET], it would be, quite simple."
+  },
+  {
+    "quipGroup":"ALPHABET_SOUP",
+    "text":"A bowl of alphabet soup has a larger vocabulary than you.",
+    "ssml":"A bowl of alphabet soup <break time=\"50ms\" /> has a larger vocabulary than you.",
```

Comment:
the minimum break time is 5-10x higher than what you have set.  Why is there even a break there to begin with?